perl/p5-perlmagick/Portfile

```diff
@@ -8,7 +8,7 @@ PortGroup           perl5 1.0
 epoch               1
 perl5.branches      5.28 5.30 5.32 5.34
 perl5.setup         PerlMagick 6.9.11-60
-revision            0
+revision            1
```

Comment:
The version of p5-perlmagick should be updated to match ImageMagick rather than increasing its revision.

Reply:
Thanks, fixed.